### PR TITLE
Add postinstall script, fix chokidar version

### PR DESCRIPTION
**Pull request by yavorsky**

### CODE REVIEW

**Comment by es128** [comment]
I like it, but have you tried it on windows?

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   },
   "scripts": {
     "prepublish": "coffee -o lib/ src/",
+    "postinstall": "if [ ! -d lib ]; then coffee -o lib/ src/; fi",
```
Comment (es128) [inline package.json]: How about `npm run prepublish` here instead of the `coffee` command?

Comment (yavorsky) [inline package.json]: @es128 :+1:  But need to modify script for windows...

Comment (es128) [inline package.json]: Perhaps something like `node -e 'if (fs.existsSync(...`

**Comment by yavorsky** [comment]
fs.exists was deprecated. I'll fix with alternative in a minute

**Comment by es128** [comment]
I know it's deprecated, but for the sake of brevity wouldn't mind if it was used here in this way.

**Comment by es128** [comment]
Are you sure the `require` statements are needed with `node -e`? It's not necessary in the REPL

**Comment by yavorsky** [comment]
@es128 necessary in eval mode.

**Comment by es128** [comment]
Ok. Too bad.

**Comment by es128** [comment]
```json
"postinstall": "node -e 'require(\"fs\").readdir(\"lib\",function(e){e&&require(\"child_process\").exec(\"npm run prepublish\")})'"
```

**Comment by es128** [comment]
This appears to be ready. Any other feedback?

**Comment by yavorsky** [comment]
I thinks it's the shortest possible cross-platform script

**Comment by paulmillr** [comment]
have you tested it on windows?

**Comment by es128** [comment]
I did just try it on windows, and it looks like the only issue is that windows requires an inversion of the double and single quotes (more specifically, the cli string argument must be wrapped in double-quotes, it doesn't understand the single quotes there)

`"postinstall": "node -e \"require('fs').readdir('lib',function(e){e&&require('child_process').exec('npm run prepublish')})\""`

@yavorsky Please set it like this in your branch and I'll try again to install from the remote git source just to be sure.

**Comment by paulmillr** [comment]
@es128 is it cool?

**Comment by es128** [comment]
Seems ok

image

**Pull request merged**